lib/src/version.dart

```diff
@@ -1,2 +1,2 @@
 // Generated code. Do not modify.
```

Comment:
Should this be in .gitignore?

Reply:
That is file is generated by build_runner to provide the version number when running `at_app --version`.